dspace/modules/api/src/main/java/org/datadryad/api/DashService.java

```diff
@@ -325,18 +327,312 @@ public int submitDashDataset(String doi) {
             responseCode = connection.getResponseCode();
 
             if(responseCode == 200 || responseCode == 201 || responseCode == 202) {
-                log.debug("package submission successful");
+                log.debug("curation activity added");
```

Comment:
This looks like a copy/paste error.... this method isn't about curation activity.
```suggestion
                log.debug("package submission successful");
```